_install-and-configure/install-opensearch/windows.md

```diff
@@ -49,13 +49,6 @@ An OpenSearch node in its default configuration (with demo certificates and user
 
 1. Run the demo batch script.
 
-   There are two ways of running the batch script:
-
-   1. Run the batch script using the Windows UI:
-
-      1. Navigate to the top directory of your OpenSearch installation and open the `opensearch-{{site.opensearch_version}}` folder.
-      1. Run the batch script by double-clicking the `opensearch-windows-install.bat` file. This opens a command prompt with an OpenSearch instance running.
```

Comment:
This is technically possible using `setx OPENSEARCH_INITIAL_ADMIN_PASSWORD <password>`, but that is essentially the same as below, and no longer one click from GUI perspective.